posts/2020/08/but-what-are-traitlets.md

```diff
@@ -0,0 +1,305 @@
+<!--
+.. title: But What are Traitelts ?
+.. slug: what-are-traitlets
+.. date: 2020-08-xx 22:00:00 UTC-00:00
```

Comment:
I think you have to put a concrete date in to not break the build. And it needs to be in the past for the post to show up.

Reply:
I've put monday, I'll try to release traitlets by then, we can also do a back to back post, if we publish that first I can also link to it when I tweet the release of traitlets 5.x